docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -133,209 +96,158 @@ code to use in your new service.
-role="download"} to the cmd/device-simple/res folder (this will
-overwrite an existing file -- that's ok).
+    ![image](EdgeX_SampleDeviceProfile_DeviceResources.png)
 
-## Rebuild your Device Service
+2.  A pre-created device profile for the random number device is provided in this documentation.  Download **[random-generator-device.yaml](random-generator-device.yaml)** and save the file to the `~/go/src/github.com/edgexfoundry/device-simple/cmd/device-simple/res` folder.
 
-Just as you did before, you are ready to build the device-simple service
--- creating the executable program that is your Device Service:
+3.  Open the random-generator-device.yaml file in a text editor. In this device profile, the device described has a deviceResource:  `randomnumber`.  Note how the association of a type to the deviceResource.  In this case, the device profile informs EdgeX that randomnumber will be a INT32.  In real world IoT situations, this deviceResource list could be extensive.  Rather than a single deviceResource, you might find this section filled with many deviceResources and each deviceResource associated to a different type.  Note also how the device profile describes a REST command (GET Random) to call to get the random number from the device service.
 
-1.  In a terminal window, change directories to the base device-simple
-    folder (containing the Makefile).
+## Configuring your Device Service
 
-2.  Build the Device Service by issuing the following command:
+Now update the configuration for the new device service.    This documentation provides a new configuration.toml file.  This configuration file:
 
-        make build
+- changes the port the service operates on so as not to conflict with other device services
+- alters the the auto event frequency, which determines when the device service collects data from the simulated device (every 10 seconds)
+- sets up the initial provisioning of the random number generating device when the service starts
 
-    ![image](EdgeX_GettingStartedSDKRebuild1.png)
+Download  **[configuration.toml](configuration.toml)** and save the file to the `~/go/src/github.com/edgexfoundry/device-simple/cmd/device-simple/res` folder (overwrite the existing configuration file).  Change the host address of the device service to your system's IP address.
 
-3.  If there are no errors, your service has now been created and is
-    available in the cmd/device-simple folder (look for the
-    device-simple file).
+!!! Warning
+    In the configuration.toml, change the host address (around line 7) to the IP address of the system host.  This allows core metadata to callback to your new device service when a new device is created.  Because the rest of EdgeX, to include core metadata, will be running in Docker, the IP address of the host system must be provide to allow metadata in Docker to call out from Docker to the new device service running on your host system.
```

Comment:
s/provide/provided

Also, I might reword that last sentence a little to include the text below between the slashes just to make it clear that the address is relative to the Docker network.

"...the IP address of the host system /on the Docker network/ must be provided..."